### PR TITLE
update toc

### PR DESCRIPTION
**Pull request by tsnow03**

### CODE REVIEW

**Comment by tsnow03** [comment]
Need to update environment.yml and lock files with hub-image environment.yml so new imports will work.

When run tutorial, still get this error for icepyx, hvplot and geoviews:
`ImportError: cannot import name 'Box' from 'bokeh.models' (/srv/conda/envs/notebook/lib/python3.10/site-packages/bokeh/models/__init__.py)`

boto3 also missing from the environment.

**Comment by tsnow03** [comment]
The quality control action is not working the way we want. I can't find any of the spelling errors it is referencing. Please ignore it for now. The rest is good to go for now. Once we get the environment updated, I will add the tutorial into the jupyterbook via the toc. It has several library errors (hvplot, geoviews, icepyx) still that should all be related to something wrong with bokeh.model.Box. 'ImportError: cannot import name 'Box' from 'bokeh.models' (/srv/conda/envs/notebook/lib/python3.10/site-packages/bokeh/models/__init__.py)'. boto3 should be the only missing package.

I'm getting `AttributeError: 'list' object has no attribute 'minimum_rotated_rectangle'` in the [IS2_cloud_data_integration Notebook](book/tutorials/IS2_cloud_data_integration.ipynb). @JessicaS11 Have you seen this icepyx Query .visualize_elevation() error I am getting? I'm using the Berkeley cloud to test the notebook and I'm wondering if it is an icepyx error or a hub issue.

**Comment by tsnow03** [comment]
I have also now disabled the Repo2Docker and Repo2DockerPR Actions since our Docker image is now coming from the hub-image repo.

File: conda/environment.yml [hunk 1]
```diff
@@ -3,24 +3,112 @@
 
 # !!! If you are developing the website locally use the existing lock files
 # for your platform (`conda create --name cryocloud --file conda-linux-64.lock`)
-name: cryocloud
+
+name: cryointhecloud
```
Comment (fperez) [inline conda/environment.yml:7]: Oh sorry! When I updated this file in the hub image repo, it didn't have a name so I made one up, and used the longer version.  But I didn't see you'd already named it, and didn't mean to override you :)

I actually like your shorter name better - let's keep your shorter one and use that instead! I can update the name in the other one. We'll be typing that often, might as well use something more concise.

**Review by fperez — APPROVED** [review]
This looks good! I suggest just reverting the env name change (I'll update the other file) and we can proceed with the merge. Might as well keep this moving quickly.

File: conda/environment.yml (hunk 1 above)
Comment (tsnow03) [reply conda/environment.yml:7]: cryocloud is perfect

**Pull request merged**